File: trail1/ab 사이에 있는 c/c-between-a-and-b.cs
using System;

public class Codetree
{  
    public static void Main()
    {
        string[] arr = Console.ReadLine().Split();
        var(a, b, c) = (int.Parse(arr[0]), int.Parse(arr[1]), int.Parse(arr[2]));
        bool flag = false;
        for (int i=a; i<=b; i++) {
            if (i % c == 0) {flag = true; break;}
        }
        Console.WriteLine(flag==true ? "YES" : "NO");
    }
}

File: trail1/ab 사이에 있는 c/c-between-a-and-b.c
#include <stdio.h>
#include <stdbool.h>

int main() {
    int a, b, c;
    scanf("%d %d %d", &a, &b, &c);
    bool flag = false;
    for (int i=a; i<=b; i++) {if(i % c == 0) {flag = true; break;}}
    const char* val = (flag==true) ? "YES" : "NO";
    printf(val);
    return 0;
}